source/reference/minio-mc-admin/mc-admin-user-info.rst

```diff
@@ -91,14 +91,25 @@ Use :mc-cmd:`mc admin user info` to view detailed user information for a user on
 
 - Replace :mc-cmd:`USERNAME <mc admin user info USERNAME>` with the username of the user to display information for.
 
-The output resembles the following:
+The output resembles the following when using the built-in MinIO Identity Provider:
```

Comment:
Maybe? (To match the `For ...` below)

```suggestion
For the MinIO internal IDentity Provider (IDP), the output resembles the following:
```

Could link, even. To [MinIO Identity Management](https://min.io/docs/minio/linux/administration/identity-access-management/minio-identity-management.html) unless there's something better?